pkg/smart/bindata.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
Rather than using go-bindata we can start using https://github.com/rakyll/statik we have started using this in our kubectl-minio plugin

Much nicer to use as well.

Reply:
I just looked into statik. It doesn't seem like a good idea for this usecase. 

Statik seems to be purpose built for a http file, i.e. streaming zipped data. Unless we are doing file streaming over the network, which requires a heap data structure with the file contents, we are better of putting this info on the stack and only loading it as needed.

Reply:
No statik can embed in the same manner as you can embed `go-bindata` - go-bindata is sort of unmaintained so we should ideally move to something which is actively maintained

https://github.com/gobuffalo/packr